### ISSUE TITLE
Jackhammer doesn't dig deepslate

### ISSUE THREAD

**Issue by 9ac0bs** [TechReborn/TechReborn#3180]
**Describe the error**
Jackhammers (basic, advanced, industrial) do not dig deep shale, basalt, netherak and endernyak. Ore from mods does not drip correctly.

**Reproduction Steps**
- Take a charged industrial jackhammer from the creative, enchant it for efficiency 5 and silk touch.
- Try to mine a stone (everything is ok)
- Try to mine a stone next to the ore from Tech Reborn (the ore will fall out)
- Try to mine a stone next to ore from Minecraft (the ore will not fall out)
- Try to mine andesite, diorite and granite (everything is ok). But for some reason smooth basalt, basalt and calcite are not mined.
- Try to mine an end stone (slowly)
- Try to mine a netherrack (slowly)
- Try to mine deepslate (slowly)
- Try to mine tuff (slowly)

**Screenshots**
Added gifs via the link for additional context

**Environment (please include the following information along with the version):**
   - Minecraft: 1.20.1
   - Mod loader: cloth 0.14.22 for 1.20.1

**Magazines**
Not required

**Additional context**
https://imgur.com/a/BKAhMNW

**Comment by xanthian** [comment]
I'm not certain but this could be the issue?

```java
	@Override
	public float getMiningSpeedMultiplier(ItemStack stack, BlockState state) {
		if (getStoredEnergy(stack) < cost) return unpoweredSpeed;
		if (ToolsUtil.JackHammerSkippedBlocks(state)) return unpoweredSpeed;

		if (state.isIn(BlockTags.STONE_ORE_REPLACEABLES)) {
			return miningSpeed;
		} else {
			return unpoweredSpeed;
		}
	}
```

If I'm understanding it correctly, if the block isnt in STONE_ORE_REPLACEABLES tag it is effectively unpowered.

The blocks in said tag are : 

    "minecraft:stone",
    "minecraft:granite",
    "minecraft:diorite",
    "minecraft:andesite"


might be better to consider using BASE_STONE_OVERWORLD & BASE_STONE_NETHER tags instead
Although that wouldnt capture end blocks

**Comment by xanthian** [comment]
Submitted a PR to resolve the vast majority of block breaking  issues.
#3181

**Comment by drcrazy** [comment]
Closed by #3181
